File: CursoFlutter/curso_dart/fundamentos/ExemploEvy.java
//package ./src/ExemploEvy

public Class ExemploEvy {
    public static final Double PI = 3.1415;

    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in); 
        System.out.println("Qual o valor do Raio: ")
        Double raio = entrada.nextDouble(); 
        Double circuferencia = PI * raio * raio;
        System.out.println("O valor da circuferencia é : " + circuferencia);
        
    }

}

File: CursoFlutter/curso_dart/fundamentos/exemplo_evy.dart
import 'dart:io';

main() {
  const PI = 3.1415;
  //print("Qual o valor do Raio: ");
  //double raio = double.parse(stdin.readLineSync()!);

  //double circuferencia = PI * raio * raio;

  //print("O valor da circuferencia é: " + circuferencia.toString());

  List nomes = ["Evy", "Wagner", "Vini", "Brayaw", "Wendell"]; 
  Map<String, String> propriedadesPc = { "Processador": "Ryzen 5" , "Placa de video": "Gtx 1060"};
  Set listaCpf = {"11711734900", "099053229927", "61561165615", "11711734900"};
  
  listaCpf.forEach((element) {print(element);});

  propriedadesPc.forEach((key, value) {
      print("Chave: " + key + " | Valor: " + value);
      });

  // for(String nome in nomes) {
  //   print("Nome  - " + nome);
  // }

}